packages/big-design/src/components/AccordionGroup/hooks/useAccordion.tsx

```diff
@@ -0,0 +1,26 @@
+import { useEffect, useState } from 'react';
+
+interface UseAccordionProps {
+  defaultExpanded?: boolean;
+}
+
+export const useAccordion = ({ defaultExpanded }: UseAccordionProps) => {
+  const [isExpanded, setisExpanded] = useState(false);
```

Comment:
Lets initialize the state to `defaultExpanded`, so that you don't need to use the `useEffect` below.

```suggestion
  const [isExpanded, setisExpanded] = useState(defaultExpanded);
```